plugins/org.locationtech.udig.tool.default.tests/src/org/locationtech/udig/tool/tests/DefaultLocaleRule.java

```diff
@@ -0,0 +1,39 @@
+package org.locationtech.udig.tool.tests;
+
+import java.util.Locale;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/** JUnit rule for taking control over the Locale. */
+public final class DefaultLocaleRule implements TestRule {

```

Comment:
I guess regarding Eclipse IP its not allowed to copy code from other projects. So it can inspire uDig but this class needs to be re-written or we add a dependency to the project to use it as a library.

Reply:
The class is quite elementary in terms of behavior and applied logic. Would it be OK if I just rename it (for example ConfigureLocaleRule) and possibly alter a couple of Comments